### DISCUSSION TITLE
PHOTOPRISM_DARKTABLE_PRESETS vs PHOTOPRISM_RAW_PRESETS

### DISCUSSION THREAD

**Discussion by cryptoluks** [photoprism/photoprism#2206] [Q&A] [answered]
Hi,

while configuring photoprism with `docker-compose`, I stumbled across the env var `PHOTOPRISM_DARKTABLE_PRESETS` which is used in all example `docker-compose` files. However, there is no reference in the official documentation nor any reference inside the code I was able to find.

Is maybe `PHOTOPRISM_RAW_PRESETS` the correct env var?

Thanks for your help.

**Comment by lastzero** [comment, marked as answer]
**Thanks for reporting this!** :+1: 

Config examples have been updated in commit https://github.com/photoprism/photoprism/commit/bfb6ab8bf72372ba824204b2404e1e2ee1662e95 (not uploaded to dl.photoprism.app yet).
